rsrc.go

```diff
@@ -1,8 +1,7 @@
-package main
+package rsrc
```

Comment:
Please don't change the import semantics of this path. It's not OK to suddenly change the meaning of `go get github.com/akavel/rsrc` for people (till now, it was building an executable; after this change, it would only build a package). Keep the `main` here, we can't change it; instead, move the extracted funcs into something like `github.com/akavel/rsrc/rsrc` or `.../rsrc/pkg/rsrc` (don't have better idea for package name in this situation unfortunately :/ do you?)